src/components/workflow/WorkflowLayout.vue

```diff
@@ -0,0 +1,203 @@
+      changes may have questions or comments. Perhaps the coding style doesn't
+      match project guidelines, the change is missing unit tests, or maybe
+      everything looks great and props are in order. Pull Requests are designed
+      to encourage and capture this type of conversation.
+    </p>
+    <p>
+      You can also continue to push to your branch in light of discussion and
+      feedback about your commits. If someone comments that you forgot to do
+      something or if there is a bug in the code, you can fix it in your branch
+      and push up the change. GitHub will show your new commits and any
+      additional feedback you may receive in the unified Pull Request view.
+    </p>
+    <v-row justify="center">
+      <image-require
+        urlImage="workflow/workflow-image4.svg"
+        altImage="Discuss and review your code"
+      />
+    </v-row>
+    <h3 class="margin-title">
+      <v-icon class="orange-color">mdi-chevron-right</v-icon> Deploy
+    </h3>
+    <p>
+      With GitHub, you can deploy from a branch for final testing in production
+      before merging to main.
+    </p>
+    <p>
+      Once your pull request has been reviewed and the branch passes your tests,
+      you can deploy your changes to verify them in production. If your branch
+      causes issues, you can roll it back by deploying the existing main branch
+      into production.
+    </p>
+    <p>
+      Different teams may have different deployment strategies. For some, it may
+      be best to deploy to a specially provisioned testing environment. For
+      others, deploying directly to production may be the better choice based on
+      the other elements in their workflow.
+    </p>
+    <v-row justify="center">
+      <image-require
+        urlImage="workflow/workflow-image5.svg"
+        altImage="Deploy"
+      />
+    </v-row>
+    <h3 class="margin-title">
+      <v-icon class="orange-color">mdi-chevron-right</v-icon> Merge
+    </h3>
+    <p>
+      Now that your changes have been verified in production, it is time to
+      merge your code into the main branch.
+    </p>
+    <p>
+      Once merged, Pull Requests preserve a record of the historical changes to
+      your code. Because they're searchable, they let anyone go back in time to
+      understand why and how a decision was made.
+    </p>
+    <v-row justify="center">
+      <image-require urlImage="workflow/workflow-image6.svg" altImage="Merge" />
+    </v-row>
+  </v-sheet>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+import ListComponent from "@/components/common/ListComponent.vue";
+import ImageRequire from "@/components/common/ImageRequire.vue";
+
+@Component({
+  name: "WorkflowLayout",
+  components: {
+    ListComponent,
+    ImageRequire,
+  },
+})
+export default class WorkflowLayout extends Vue {}
+</script>
+
+<style lang="scss" scoped>
+.text {
```

Comment:
@JuanJosePG BEM methodology and maybe you can create a component to don't repeat code